app/helpers/application_helper/button/cockpit_console.rb

```diff
@@ -15,7 +15,7 @@ def on?
     @record.ready_condition_status == 'True' if @record.respond_to?(:ready_condition_status) # Container status
   end
 
-  def platform_supported?
-    @record.respond_to?(:current_state) && @record.platform.downcase != 'windows'
+  def platform_supported?(record_type)
+    @record.platform.downcase != 'windows' if record_type == 'VM'
```

Comment:
Don't you need to return true when record_type != VM

Reply:
@bmclaughlin just wanted to make sure you saw my comment above.

Reply:
@petervo Thanks, I had missed your previous comment.  When I checked in the UI the button is enabled.  The method name versus use is confusing, I'll update it to `platform_unsupported?(record_type)`  Thanks!